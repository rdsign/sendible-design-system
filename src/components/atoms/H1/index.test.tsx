import { create } from 'react-test-renderer';
import { H1 } from './index';

describe('Icon', () => {
    const component = <H1 />;

    test('should match snapshot', () => {
        const snapshot = create(component);
        expect(snapshot).toMatchSnapshot();
    });
});
