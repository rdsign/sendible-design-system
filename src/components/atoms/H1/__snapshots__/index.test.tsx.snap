// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon should match snapshot 1`] = `
<h1>
  Ol√°
</h1>
`;
